Cargo.toml

```diff
@@ -9,14 +9,21 @@ readme = "README.md"
 repository = "https://github.com/maidsafe/crust"
 version = "0.31.0"
 
+[[example]]
```

Comment:
doesn't this normally come at the end of the file ? Usually we are more interested to quickly know the deps, and those should be at the top.

Reply:
`cargo edit` reformatted it this way when I did `cargo add mio-extras`. Unless we don't want to use `cargo edit` anymore.